server/database/queries/teacher/check-deleted-quiz.ts

```diff
@@ -0,0 +1,4 @@
+import connections from '../../config/connections';
+
+export default (quizId: String, teacherId:Number) => connections
```

Comment:
There's no need for this query, we can check if the quiz has been deleted or not by trying to delete it again